clang-tools-extra/clang-tidy/ClangTidy.cpp

```diff
@@ -458,7 +458,6 @@ ClangTidyASTConsumerFactory::createASTConsumer(
   if (!AnalyzerOptions.CheckersAndPackages.empty()) {
     setStaticAnalyzerCheckerOpts(Context.getOptions(), AnalyzerOptions);
     AnalyzerOptions.AnalysisDiagOpt = PD_NONE;
-    AnalyzerOptions.eagerlyAssumeBinOpBifurcation = true;
```

Comment:
Is this a functionality loss from Tidy's perspective? AFAIK, there is no way to set CSA analyser engine options when invoked through Tidy.

Reply:
Clang-Tidy can set CSA options, I recently added support for it for `--verify-config` in #109523, where I had checked which syntax it has, and if the syntax worked beforehand.

Reply:
No, this is not functionality loss from Tidy's perspective, because this data member is dead, its value doesn't influence anything. The `eagerly-assume` feature is controlled by `AnalyzerOptions.ShouldEagerlyAssume` (which is true by default).